Post/forms.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+from .models import Post
+
+
+class AddAPostForm(forms.ModelForm):
```

Comment:
maybe AddaPostForm will break the capital letters sequence, but that's only one opinion, to just AddPostForm

Reply:
I changed it to CreatePostForm. Is it better? @maayanhd